content/api/local-installation.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Local installation
+---
+
+Installing our code locally will allow your editor to extract type definitions from the `Robot` class and its boards, as well as allowing auto-complete (also known as [IntelliSense](https://en.wikipedia.org/wiki/IntelliSense)).
```

Comment:
While correct, the phrasing here feels quite technical. I expect that inexperienced readers would be confused by this (what's a "type definition"?).

Unless we're deliberately targetting advanced users, we should probably keep this fairly high level.

(I'd be completely open to having an "Advanced" section which explains that we have added type annotations compatible with `mypy`)